src/components/home.js

```diff
@@ -0,0 +1,66 @@
+function Home() {
+  return (
+    <div style={wrapperStyle}>
+      <div style={headerStyle}>
+        <h2>Protect Your Data. Build Your Future.</h2>
+        <p style={headerDetailStyle}>
+          The OpenTDF project is an open set of tools and services that allows
+          you to protect your data everywhere it goes.
+        </p>
+        <a
+          style={getStartedButtonStyle}
+          href="https://github.com/opentdf/opentdf/tree/main/quickstart#opentdf-quickstart"
+        >
+          Get Started
+        </a>
+        <a style={learnMoreButtonStyle} href="https://github.com/orgs/opentdf/">
+          Learn More
+        </a>
+      </div>
+    </div>
+  );
+}
+
+const buttonStyle = {
+  appearance: "button",
+  WebkitAppearance: "button",
+  MozAppearance: "button",
+  textDecoration: "none",
+  padding: "6px 18px",
+  fontWeight: "bold",
+  border: "white",
+  borderStyle: "solid",
+  borderWidth: "thin",
+  marginTop: "10px",
+};
+
+const getStartedButtonStyle = {
+  marginRight: "2vw",
+  backgroundColor: "white",
+  color: "#04A777",
+  ...buttonStyle,
+};
+
+const learnMoreButtonStyle = {
+  color: "white",
+  backgroundColor: "transparent",
+  ...buttonStyle,
```

Comment:
```suggestion
  "-webkit-appearance": "none",
  ...buttonStyle,
```

https://stackoverflow.com/questions/33328102/safari-shows-submit-button-background-color-whitish

I think this is stolen from a safari style reset stylesheet. It looks like it also resets the margins, so you might want to add those back, but it looks fine so long as you only reset one button, not both.